packages/react-native-web/src/modules/applyLayout/index.js

```diff
@@ -55,10 +55,10 @@ const observe = instance => {
     node._onLayoutId = id;
     resizeObserver.observe(node);
   } else {
-    const id = guid();
```

Comment:
I think this also fixes a bug. This `id` is never assigned in the registry (L50 is where the first `id` is created), and so the instance won't be removed in L65

Reply:
🤦‍♂️ correct